akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/TestProbeSpec.scala

```diff
@@ -10,6 +10,8 @@ import org.scalatest.WordSpecLike
 
 class TestProbeSpec extends ScalaTestWithActorTestKit with WordSpecLike {
```

Comment:
would be good to exercise the javadsl also, in TestProbeTest, not that I changed that one to a real test in https://github.com/akka/akka/pull/25833

Reply:
Agree, and I've not written java for so long the test makes no sense as to where it is even sending any messages for me to attempt to test these. I'll take a third stab...